### DISCUSSION TITLE
Mamba

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#28] [General]
Discuss things related to Mamba here!

**Comment by Jenildabhi5103** [comment]
How can I start with Mamba documentation? Can anyone help me with this?

**Reply by itsnotmik** [reply]
Id suggest starting with the method of installing mamba + the things we are currently installing with mamba

Mamba installation is through Conda-forge docker image

Requirements installed through mamba found in the requirements.txt
